<!DOCTYPE html>
<html>

  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">

    <title>Murray Patterson</title>
  </head>

  <body style="max-width:960px; margin-left:auto; margin-right:auto">

    <table>
      <tr>

	<td align="center" width="40%">
	  <img src="20200911SAT_Murray_Patterson_PSF.jpg" width="50%">
	</td>

	<td >
	  <p><font size="6">Murray Patterson</font></p>

	  <p>
	    Assistant Professor
	  </p>

	  <p>
	    Department of Computer Science
	    <br>
	    Georgia State University
	    <br>
	    1 Park Place
	    <br>
	    Atlanta, Georgia 30303 (USA)
	  </p>

	  <p>
	    office: Room 636
	    <br>
	    tel: (+1) 475 330 0978
	    <br>
	    email: <tt>mpatterson<font color="#6e6e6e">[at]</font>cs<font color="#6e6e6e">[dot]</font>gsu<font color="#6e6e6e">[dot]</font>edu</tt>
	  </p>
	</td>

      </tr>
    </table>
    <br>

    <p>
      Since the fall of 2020, I am a tenure track Assistant Professor
      of the Bioinformatics Cluster in
      the <a href="https://www.csds.gsu.edu/">Computer Science
      Department</a> at <a href="https://www.gsu.edu/">Georgia State
      University</a>, Atlanta, Georgia.
    </p>
    
    <p>
      In the 2019&mdash;2020 academic year, I was a Visiting Assistant
      Professor of Computer Science in
      the <a href="https://www.fairfield.edu/undergraduate/academics/schools-and-colleges/school-of-engineering/index.html">School
      of Engineering</a>
      at <a href="https://www.fairfield.edu">Fairfield University</a>.
    </p>

    <p>
      From the summer of 2016 until the summer of 2019, I was a
      postdoctoral fellow in
      the <a href="https://algolab.eu/">Experimental Algorithmics
      Lab</a> at the <a href="http://www.unimib.it">University of
      Milano-Bicocca</a>
      with <a href="http://gianluca.dellavedova.org">Gianluca Della
      Vedova</a>
      and <a href="https://algolab.eu/people/bonizzoni">Paola
      Bonizzoni</a>.  My project was on haplotyping and cancer
      phylogenies.
    </p>

    <p>
      From the spring of 2014 I did a 2-year postdoctoral fellowship
      at the <a href="http://lbbe.univ-lyon1.fr">LBBE</a> as part of
      the Ancestrome Project with Daniel Kahn
      and <a href="https://lbbe.univ-lyon1.fr/fr/annuaires-des-membres/daubin-vincent">Vincent
      Daubin</a>.  My project was on the evolution of metabolic
      networks from the perspective of the enzymes that catalyse
      reactions, but also of the structure of the network itself.
    </p>

    <p>
      From the beginning of 2012 I was
      an <a href="http://www.ercim.eu/">ERCIM fellow</a> with a 2-year
      Marie
      Curie <a href="https://fellowship.ercim.eu/current_fellows">ABCDE
      fellowship</a>.  I held my first year of the fellowship with
      the <a href="http://www.inria.fr">INRIA</a> at
      the <a href="http://lbbe.univ-lyon1.fr">Laboratoire de
      Biom&eacutetrie et Biologie &Eacutevolutive (LBBE)</a> of
      the <a href="http://www.univ-lyon1.fr">Universit&eacute de
      Lyon</a> in France
      with <a href="https://lbbe.univ-lyon1.fr/fr/annuaires-des-membres/tannier-eric">Eric
      Tannier</a>.  I held my second year in
      the <a href="https://www.cwi.nl/research/groups/life-sciences-and-health">Life
      Sciences Group</a> at the <a href="https://www.cwi.nl">CWI</a>
      with <a href="http://homepages.cwi.nl/~as/">Alexander
      Schoenhuth</a>
      and <a href="https://www.cs.hhu.de/en/research-groups/algorithmic-bioinformatics">Gunnar
      Klau</a>.
    </p>

    <br><h1>Teaching</h1><hr><br>

    <p>
      <h3>Spring 2023</h3>
      <ul>
	<li> a senior year /
	  masters-level <a href="spring2023/syllabus_4710.pdf">course
	  on artificial intellgence</a>.
      </ul>
    </p>

    <p>
      <h3>Fall 2022</h3>
      <ul>
	<li> a senior year /
	  masters-level <a href="fall2022/syllabus_4330.pdf">programming
	  language concepts</a> course.
      </ul>
    </p>

    <p>
      <h3>Spring 2022</h3>
      <ul>
	<li> a senior year /
	  masters-level <a href="spring2022/syllabus_4330.pdf">programming
	  language concepts</a> course.
      </ul>
    </p>

    <p>
      <h3>Fall 2021</h3>
      <ul>
	<li> a senior year /
	  masters-level <a href="fall2021/syllabus_4810.pdf">course on
	  database management systems</a>.
      </ul>
    </p>
    
    <p>
      <h3>Spring 2021</h3>
      <ul>
	<li> a senior year /
	  masters-level <a href="spring2021/syllabus_4330.pdf">programming
	  language concepts</a> course.
      </ul>
    </p>
    
    <p>
      <h3>Fall 2020</h3>
      <ul>
	<li> a senior year /
	  masters-level <a href="fall2020/syllabus_4330.pdf">programming
	  language concepts</a> course.
      </ul>
    </p>

    <p>
      <h3>Spring 2020</h3>
      <ul>
	<li>
	  an <a href="spring2020/syllabus_131_cardinal.pdf">introductory programming course</a> in Java, and
	<li>
	  a junior year <a href="spring2020/syllabus_354.pdf">theory of programming languages course</a> using Haskell and Prolog.
      </ul>
    </p>

    <p>
      <h3>Fall 2019</h3>
      <ul>
	<li>
	  an <a href="fall2019/syllabus_131.pdf">introductory
	    programming course</a> in Java,
	<li>
	  an <a href="fall2019/syllabus_231.pdf">intermediate
	  programming course</a>, focusing on the object-oriented
	  features of Java, and
	<li>
	  a senior year / masters-level
	  <a href="fall2019/syllabus_402.pdf">course
	    on database management systems</a>.
      </ul>

    <br><h1>Publications</h1><hr><br>

    <p>
      View also
      at <a href="http://scholar.google.ca/citations?user=49aQ7scAAAAJ&hl=en">Google
      Scholar</a>, <a href="http://www.informatik.uni-trier.de/~ley/pers/hc/p/Patterson:Murray.html">DBLP</a>
      or see my full <a href="cv.pdf">CV</a>, among a list
      of supporting <a href="documents.html">documents</a>
    </p>
    <br>

    <!-- content -->
<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN">
<html>
<head><meta name="generator" content="Pybtex">
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<title>Bibliography</title>
</head>
<body>
<dl>
<h2>Journal Articles</h2>
<dt>1.</dt>
<dd>Sarwan Ali, Babatunde Bello, and Murray Patterson.
Solvent accessibility of coronaviridae spike proteins through the lens of information gain.
<em>MDPI J — Multidisciplinary Scientific Journal</em>, 2023.
To appear.</dd>
<dt>2.</dt>
<dd>Sarwan Ali, Bikram Sahoo, Alexander Zelikovsky, Pin-Yu Chen, and Murray Patterson.
Benchmarking machine learning robustness in covid-19 genome sequence classification.
<em>Scientific Reports</em>, 13(1):4154, 2023.
<a href="https://doi.org/10.1038/s41598-023-31368-3">doi:10.1038/s41598-023-31368-3</a>.</dd>
<dt>3.</dt>
<dd>Prakash Chourasia, Sarwan Ali, Simone Ciccolella, Gianluca&nbsp;Della Vedova, and Murray Patterson.
<span class="bibtex-protected">Reads2Vec</span>: <span class="bibtex-protected">E</span>fficient embedding of raw high-throughput sequencing reads data.
<em>Journal of Computational Biology</em>, 2022.
<a href="https://doi.org/10.1089/cmb.2022.0424">doi:10.1089/cmb.2022.0424</a>.</dd>
<dt>4.</dt>
<dd>Sarwan Ali, Babatunde Bello, Zahra Tayebi, and Murray Patterson.
Characterizing <span class="bibtex-protected">SARS-CoV-2</span> spike sequences based on geographical location.
<em>Journal of Computational Biology</em>, 2022.
<a href="https://doi.org/10.1089/cmb.2022.0391">doi:10.1089/cmb.2022.0391</a>.</dd>
<dt>5.</dt>
<dd>Sarwan Ali, Bikram Sahoo, Muhammad&nbsp;Asad Khan, Alexander Zelikovsky, Imdad&nbsp;Ullah Khan*, and Murray Patterson*.
Efficient approximate kernel based spike sequence classification.
<em>Transactions on Computational Biology and Bioinformatics</em>, 2022.
To appear.
Preprint at: <a href="https://arxiv.org/abs/2209.04952">https://arxiv.org/abs/2209.04952</a>.</dd>
<dt>6.</dt>
<dd>Kaustubh Khandai, Cristian Navarro-Martinez, Brendan Smith, Rebecca Buonopane, S.&nbsp;Ashley Byun*, and Murray Patterson*.
Determining significant correlation between pairs of extant characters in a small parsimony framework.
<em>Journal of Computational Biology</em>, 29(10):1132&ndash;1154, 2022.
<a href="https://doi.org/10.1089/cmb.2022.0141">doi:10.1089/cmb.2022.0141</a>.</dd>
<dt>7.</dt>
<dd>Sarwan Ali, Babatunde Bello, Prakash Chourasia, Ria&nbsp;Thazhe Punathil, Yijing Zhou, and Murray Patterson.
<span class="bibtex-protected">PWM2Vec</span>: <span class="bibtex-protected">A</span>n efficient embedding approach for viral host specification from coronavirus spike sequences.
<em>MDPI Biology</em>, 11(3):418, 2022.
<a href="https://doi.org/10.3390/biology11030418">doi:10.3390/biology11030418</a>.</dd>
<dt>8.</dt>
<dd>Sarwan Ali, Yijing Zhou, and Murray Patterson.
Efficient analysis of <span class="bibtex-protected">COVID-19</span> clinical data using machine learning models.
<em>Medical &amp; Biological Engineering &amp; Computing</em>, 60(7):1881&ndash;1896, 2021.
<a href="https://doi.org/10.1007/s11517-022-02570-8">doi:10.1007/s11517-022-02570-8</a>.</dd>
<dt>9.</dt>
<dd>Zahra Tayebi, Sarwan Ali, and Murray Patterson.
Robust representation and efficient feature selection allows for effective clustering of <span class="bibtex-protected">SARS-CoV-2</span> variants.
<em>Algorithms</em>, 14(12):348, 2021.
<a href="https://doi.org/10.3390/a14120348">doi:10.3390/a14120348</a>.</dd>
<dt>10.</dt>
<dd>Sarwan Ali, Simone Ciccolella, Lorenzo Lucarella, Gianluca&nbsp;Della Vedova, and Murray Patterson.
Simpler and faster development of tumor phylogeny pipelines.
<em>Journal of Computational Biology</em>, 28(11):1142&ndash;1155, 2021.
<a href="https://doi.org/10.1089/cmb.2021.0271">doi:10.1089/cmb.2021.0271</a>.</dd>
<dt>11.</dt>
<dd>Andrew Melnyk, Fatemeh Mohebbi, Sergey Knyazev, Bikram Sahoo, Roya Hosseini, Pavel Skums, Alex Zelikovsky*, and Murray Patterson*.
From alpha to zeta: <span class="bibtex-protected">I</span>dentifying variants and subtypes of <span class="bibtex-protected">SARS-CoV-2</span> via clustering.
<em>Journal of Computational Biology</em>, 28(11):1113&ndash;1129, 2021.
<a href="https://doi.org/10.1089/cmb.2021.0302">doi:10.1089/cmb.2021.0302</a>.</dd>
<dt>12.</dt>
<dd>Simone Ciccolella*, Murray Patterson*, Paola Bonizzoni, and Gianluca&nbsp;Della Vedova.
Effective clustering for single cell sequencing cancer data.
<em>IEEE Journal of Biomedical and Health Informatics</em>, 25(11):4068&ndash;4078, 2021.
<a href="https://doi.org/10.1109/JBHI.2021.3081380">doi:10.1109/JBHI.2021.3081380</a>.</dd>
<dt>13.</dt>
<dd>Simone Ciccolella, Mauricio Soto, Murray&nbsp;D. Patterson, Gianluca&nbsp;Della Vedova, Iman Hajirasouliha, and Paola Bonizzoni.
Gpps: <span class="bibtex-protected">A</span>n <span class="bibtex-protected">ILP</span>-based approach for inferring cancer progression with mutation losses from single cell data.
<em>BMC Bioinformatics</em>, 21(1):413, 2020.
<a href="https://doi.org/10.1186/s12859-020-03736-7">doi:10.1186/s12859-020-03736-7</a>.</dd>
<dt>14.</dt>
<dd>Simone Ciccolella, Camir Ricketts, Mauricio&nbsp;Soto Gomez, Murray Patterson, Dana Silverbush, Paola Bonizzoni, Iman Hajirasouliha, and Gianluca&nbsp;Della Vedova.
Inferring cancer progression from single-cell sequencing while allowing mutation losses.
<em>Bioinformatics</em>, 37(3):326&ndash;333, 2020.
<a href="https://doi.org/10.1093/bioinformatics/btaa722">doi:10.1093/bioinformatics/btaa722</a>.</dd>
<dt>15.</dt>
<dd>Raffaella Rizzi, Stefano Beretta, Murray Patterson, Yuri Pirola, Marco Previtali, Gianluca&nbsp;Della Vedova, and Paola Bonizzoni.
Overlap graphs and de <span class="bibtex-protected">B</span>ruijn graphs: data structures for de novo genome assembly in the big data era.
<em>Quantitative Biology</em>, 7(4):278&ndash;292, 2019.
<a href="https://doi.org/10.1007/s40484-019-0181-x">doi:10.1007/s40484-019-0181-x</a>.</dd>
<dt>16.</dt>
<dd>Stefano Beretta*, Murray Patterson*, Simone Zaccaria, Gianluca&nbsp;Della Vedova, and Paola Bonizzoni.
Hap<span class="bibtex-protected">CHAT</span>: <span class="bibtex-protected">A</span>daptive haplotype assembly for efficiently leveraging high coverage in long reads.
<em>BMC Bioinformatics</em>, 19(1):252, 2018.
<a href="https://doi.org/10.1186/s12859-018-2253-8">doi:10.1186/s12859-018-2253-8</a>.</dd>
<dt>17.</dt>
<dd>Wandrille Duchemin, Yoann Anselmetti, Murray Patterson, Yann Ponty, S<span class="bibtex-protected">è</span>verine B<span class="bibtex-protected">é</span>rard, Cedric Chauve, Celine Scornavacca, Vincent Daubin, and Eric Tannier.
De<span class="bibtex-protected">C</span>o<span class="bibtex-protected">STAR</span>: <span class="bibtex-protected">R</span>econstructing the ancestral organization of genes or genomes using reconciled phylogenies.
<em>Genome Biology and Evolution</em>, 9(5):1312&ndash;1319, 2017.
<a href="https://doi.org/10.1093/gbe/evx069">doi:10.1093/gbe/evx069</a>.</dd>
<dt>18.</dt>
<dd>Andrea Bracciali, Marco Aldinucci, Murray Patterson, Tobias Marschall, Nadia Pisanti, Ivan Merelli, and Massimo Torquati.
P<span class="bibtex-protected">W</span>hats<span class="bibtex-protected">H</span>ap: <span class="bibtex-protected">E</span>fficient haplotyping for future generation sequencing.
<em>BMC Bioinformatics</em>, 17(11):342, 2016.
<a href="https://doi.org/10.1186/s12859-016-1170-y">doi:10.1186/s12859-016-1170-y</a>.</dd>
<dt>19.</dt>
<dd>Murray Patterson*, Tobias Marschall*, Nadia Pisanti, Leo van Iersel, Leen Stougie, Gunnar&nbsp;W. Klau&dagger;, and Alexander Schoenhuth&dagger;.
Whats<span class="bibtex-protected">H</span>ap: <span class="bibtex-protected">W</span>eighted haplotype assembly for future-generation sequencing reads.
<em>Journal of Computational Biology</em>, 22(6):498&ndash;509, 2015.
<a href="https://doi.org/10.1089/cmb.2014.0157">doi:10.1089/cmb.2014.0157</a>.</dd>
<dt>20.</dt>
<dd>Murray Patterson, Gergely Sz<span class="bibtex-protected">ö</span>ll<span class="bibtex-protected">ő</span>si, Vincent Daubin, and Eric Tannier.
Lateral gene transfer, rearrangement, reconciliation.
<em>BMC Bioinformatics</em>, 14(15):S4, 2013.
<a href="https://doi.org/10.1186/1471-2105-14-S15-S4">doi:10.1186/1471-2105-14-S15-S4</a>.</dd>
<dt>21.</dt>
<dd>Mohammed El-Kebir*, Tobias Marschall*, Inken Wohlers*, Murray Patterson, Jaap Heringa, Alexander Schoenhuth, and Gunnar&nbsp;W. Klau.
Mapping proteins in the presence of paralogs using units of coevolution.
<em>BMC Bioinformatics</em>, 14(15):S18, 2013.
<a href="https://doi.org/10.1186/1471-2105-14-S15-S18">doi:10.1186/1471-2105-14-S15-S18</a>.</dd>
<dt>22.</dt>
<dd>J<span class="bibtex-protected">á</span>n Ma<span class="bibtex-protected">ň</span>uch*, Murray Patterson*, Roland Wittler*, Cedric Chauve, and Eric Tannier.
Linearization of ancestral multichromosomal genomes.
<em>BMC Bioinformatics</em>, 13(19):S11, 2012.
<a href="https://doi.org/10.1186/1471-2105-13-S19-S11">doi:10.1186/1471-2105-13-S19-S11</a>.</dd>
<dt>23.</dt>
<dd>J<span class="bibtex-protected">á</span>n Ma<span class="bibtex-protected">ň</span>uch*, Murray Patterson*, and Cedric Chauve.
Hardness results on the gapped consecutive-ones property problem.
<em>Discrete Applied Mathematics</em>, 160(18):2760&ndash;2768, 2012.
<a href="https://doi.org/10.1016/j.dam.2012.03.019">doi:10.1016/j.dam.2012.03.019</a>.</dd>
<dt>24.</dt>
<dd>J<span class="bibtex-protected">á</span>n Ma<span class="bibtex-protected">ň</span>uch* and Murray Patterson*.
The complexity of the gapped consecutive-ones property problem for matrices of bounded maximum degree.
<em>Journal of Computational Biology</em>, 18(9):1243&ndash;1253, 2011.
<a href="https://doi.org/10.1089/cmb.2011.0128">doi:10.1089/cmb.2011.0128</a>.</dd>
<dt>25.</dt>
<dd>Roland Wittler, J<span class="bibtex-protected">á</span>n Ma<span class="bibtex-protected">ň</span>uch*, Murray Patterson*, and Jens Stoye.
Consistency of sequence-based gene clusters.
<em>Journal of Computational Biology</em>, 18(9):1023&ndash;1039, 2011.
<a href="https://doi.org/10.1089/cmb.2011.0083">doi:10.1089/cmb.2011.0083</a>.</dd>
<h2>Conference Proceedings</h2>
<dt>26.</dt>
<dd>Sarwan Ali, Babatunde Bello, Prakash Chourasia, Ria&nbsp;Thazhe Punathil, Pin-Yu Chen, Imdad&nbsp;Ullah Khan, and Murray Patterson.
Virus2<span class="bibtex-protected">V</span>ec: <span class="bibtex-protected">V</span>iral sequence classification using machine learning.
In <em>the 2023 Conference on Health, Inference, and Learning (CHIL, Cambridge, MA, USA, 2023)</em>. 2023.
To appear.</dd>
<dt>27.</dt>
<dd>Sarwan Ali, Prakash Chourasia, and Murray Patterson.
When biology has chemistry: <span class="bibtex-protected">S</span>olubility and drug subcategory prediction using <span class="bibtex-protected">SMILES</span> strings.
In <em>the 11th International Conference on Learning Representations (ICLR, Kigali, Rwanda, 2023)</em>, Tiny Papers Track. 2023.
To appear.</dd>
<dt>28.</dt>
<dd>Prakash Chourasia*, Zahra Tayebi*, Sarwan Ali*, and Murray Patterson.
Empowering pandemic response with federated learning for protein sequence data analysis.
In <em>the 2023 International Joint Conference on Neural Networks (IJCNN, Queensland, Australia, 2023)</em>. 2023.
To appear.</dd>
<dt>29.</dt>
<dd>Sarwan Ali*, Taslim Murad*, and Murray Patterson.
Pcd2<span class="bibtex-protected">V</span>ec: <span class="bibtex-protected">A</span> poisson correction distance based approach for viral host classification.
In <em>the 2023 International Joint Conference on Neural Networks (IJCNN, Queensland, Australia, 2023)</em>. 2023.
To appear.
Preprint at: <a href="https://doi.org/10.48550/arXiv.2304.06731">https://doi.org/10.48550/arXiv.2304.06731</a>.</dd>
<dt>30.</dt>
<dd>Sarwan Ali, Usama Sardar, Murray Patterson, and Imdad&nbsp;Ullah Khan.
Bio<span class="bibtex-protected">S</span>equence2<span class="bibtex-protected">V</span>ec: <span class="bibtex-protected">E</span>fficient embedding generation for biological sequences.
In <em>the 27th Pacific-Asia Conference on Knowledge Discovery and Data Mining (PAKDD, Osaka, Japan, 2023)</em>. 2023.
To appear.
Preprint at: <a href="https://doi.org/10.48550/arXiv.2304.00291">https://doi.org/10.48550/arXiv.2304.00291</a>.</dd>
<dt>31.</dt>
<dd>Prakash Chourasia, Sarwan Ali, and Murray Patterson.
Informative initialization and kernel selection improves <span class="bibtex-protected">t-SNE</span> for biological sequences.
In <em>the 2022 IEEE International Conference on Big Data (IEEE BigData, Osaka, Japan, 2022)</em>. 2022.
To appear.
Preprint at: <a href="https://arxiv.org/abs/2211.09263">https://arxiv.org/abs/2211.09263</a>.</dd>
<dt>32.</dt>
<dd>Sarwan Ali, Taslim Murad, and Murray Patterson.
<span class="bibtex-protected">PSSM2Vec</span>: <span class="bibtex-protected">A</span> compact alignment-free embedding approach for coronavirus spike sequence classification.
In <em>the 29th International Conference on Neural Information Processing (ICONIP, Indore, India, 2022)</em>, volume 1794 of Springer CCIS, 420&ndash;432. 2023.
<a href="https://doi.org/10.1007/978-981-99-1648-1_35">doi:10.1007/978-981-99-1648-1_35</a>.</dd>
<dt>33.</dt>
<dd>Taslim Murad*, Prakash Chourasia*, Sarwan Ali*, and Murray Patterson.
Hashing2<span class="bibtex-protected">V</span>ec: <span class="bibtex-protected">F</span>ast embedding generation for <span class="bibtex-protected">SARS-CoV-2</span> spike sequence classification.
In <em>the 14th Asian Conference on Machine Learning (ACML, Hyderabad, India, 2022)</em>, PMLR. 2022.
To appear.</dd>
<dt>34.</dt>
<dd>Prakash Chourasia, Sarwan Ali, Simone Ciccolella, Gianluca&nbsp;Della Vedova, and Murray Patterson.
Clustering <span class="bibtex-protected">SARS-CoV-2</span> variants from raw high-throughput sequencing reads data.
In <em>the 11th International Conference on Computational Advances in Bio and medical Sciences (ICCABS, Virtual Conference, 2021)</em>, volume 13254 of Springer LNBI, 133&ndash;148. 2022.
<a href="https://doi.org/10.1007/978-3-031-17531-2_11">doi:10.1007/978-3-031-17531-2_11</a>.</dd>
<dt>35.</dt>
<dd>Sarwan Ali, Taslim Murad, Prakash Chourasia, and Murray Patterson.
Spike2<span class="bibtex-protected">S</span>ignal: <span class="bibtex-protected">C</span>lassifying coronavirus spike sequences with deep learning.
In <em>the 8th IEEE International Conference on Big Data Service and Applications (IEEE BDS, San Francisco Bay Area, USA, 2022)</em>, 81&ndash;88. 2022.
<a href="https://doi.org/10.1109/BigDataService55688.2022.00020">doi:10.1109/BigDataService55688.2022.00020</a>.</dd>
<dt>36.</dt>
<dd>Sarwan Ali, Bikram Sahoo, Naimat Ullah, Alex Zelikovsky, Murray Patterson*, and Imdad&nbsp;Ullah Khan*.
A <em>k</em>-mer based approach for <span class="bibtex-protected">SARS-CoV-2</span> variant identification.
In <em>the 17th International Symposium on Bioinformatics Research and Applications (ISBRA, Shenzhen, China, 2021)</em>, volume 13064 of LNCS, 153&ndash;164. 2021.
<a href="https://doi.org/10.1007/978-3-030-91415-8_14">doi:10.1007/978-3-030-91415-8_14</a>.</dd>
<dt>37.</dt>
<dd>Brendan Smith, Cristian Navarro-Martinez, Rebecca Buonopane, S.&nbsp;Ashley Byun*, and Murray Patterson*.
Correlated evolution in the small parsimony framework.
In <em>the 17th International Symposium on Bioinformatics Research and Applications (ISBRA, Shenzhen, China, 2021)</em>, volume 13064 of LNCS, 608&ndash;619. 2021.
<a href="https://doi.org/10.1007/978-3-030-91415-8_51">doi:10.1007/978-3-030-91415-8_51</a>.</dd>
<dt>38.</dt>
<dd>Sarwan Ali and Murray Patterson.
Spike2<span class="bibtex-protected">V</span>ec: <span class="bibtex-protected">A</span>n efficient and scalable embedding approach for <span class="bibtex-protected">COVID-19</span> spike sequences.
In <em>the 2021 IEEE International Conference on Big Data (IEEE BigData, Virtual Conference, 2021)</em>, 1533&ndash;1530. 2021.
<a href="https://doi.org/10.1109/BigData52589.2021.9671848">doi:10.1109/BigData52589.2021.9671848</a>.</dd>
<dt>39.</dt>
<dd>Sarwan Ali, Tamkanat-E-Ali, Muhammad&nbsp;Asad Khan, Imdad&nbsp;Ullah Khan, and Murray Patterson.
Effective and scalable clustering of <span class="bibtex-protected">SARS-CoV-2</span> sequences.
In <em>the 5th International Conference on Big Data Research (ICBDR, Tokyo, Japan, 2021)</em>, 42&ndash;49. 2021.
<a href="https://doi.org/10.1145/3505745.3505752">doi:10.1145/3505745.3505752</a>.</dd>
<dt>40.</dt>
<dd>Andrew Melnyk, Fatemeh Mohebbi, Sergey Knyazev, Bikram Sahoo, Roya Hosseini, Pavel Skums, Alex Zelikovsky*, and Murray Patterson*.
Clustering based identification of <span class="bibtex-protected">SARS-CoV-2</span> subtypes.
In <em>the 10th International Conference on Computational Advances in Bio and Medical Sciences (ICCABS, Virtual Conference, 2020)</em>, volume 12686 of LNCS, 127&ndash;141. 2021.
<a href="https://doi.org/10.1007/978-3-030-79290-9_11">doi:10.1007/978-3-030-79290-9_11</a>.</dd>
<dt>41.</dt>
<dd>Simone Ciccolella*, Murray Patterson*, Paola Bonizzoni, and Gianluca&nbsp;Della Vedova.
Effective clustering for single cell sequencing cancer data.
In <em>the 10th ACM International Conference on Bioinformatics, Computational Biology and Health Informatics (ACM-BCB, Niagara Falls, NY, USA, 2019)</em>, ACM, 437&ndash;446. 2019.
<a href="https://doi.org/10.1145/3307339.3342149">doi:10.1145/3307339.3342149</a>.</dd>
<dt>42.</dt>
<dd>Giulia Bernardini*, Paola Bonizzoni*, Gianluca&nbsp;Della Vedova*, and Murray Patterson*.
A rearrangement distance for fully-labelled trees.
In <em>the 30th Annual Symposium on Combinatorial Pattern Matching (CPM, Pisa, Italy, 2019)</em>, volume 128 of LIPIcs, 1&ndash;15. 2019.
<a href="https://doi.org/10.4230/LIPIcs.CPM.2019.28">doi:10.4230/LIPIcs.CPM.2019.28</a>.</dd>
<dt>43.</dt>
<dd>Gianluca&nbsp;Della Vedova*, Murray Patterson*, Raffaella Rizzi*, and Mauricio Soto*.
Character-based phylogeny construction and its application to tumor evolution.
In <em>the 13th Conference on Computability in Europe (CiE, Turku, Finland, 2017)</em>, volume 10307 of LNCS, 3&ndash;13. 2017.
<a href="https://doi.org/10.1007/978-3-319-58741-7_1">doi:10.1007/978-3-319-58741-7_1</a>.</dd>
<dt>44.</dt>
<dd>Marco Aldinucci, Andrea Bracciali, Tobias Marschall, Murray Patterson, Nadia Pisanti, and Massimo Torquati.
High-performance haplotype assembly.
In <em>the 11th International Meeting on Computational Intelligence Methods for Bioinformatics and Biostatistics (CIBB, Cambridge, UK, 2014)</em>, volume 8623 of LNCS, 245&ndash;258. 2015.
<a href="https://doi.org/10.1007/978-3-319-24462-4_21">doi:10.1007/978-3-319-24462-4_21</a>.</dd>
<dt>45.</dt>
<dd>Murray Patterson*, Tobias Marschall*, Nadia Pisanti, Leo van Iersel, Leen Stougie, Gunnar&nbsp;W. Klau&dagger;, and Alexander Schoenhuth&dagger;.
Whats<span class="bibtex-protected">H</span>ap: <span class="bibtex-protected">H</span>aplotype assembly for future-generation sequencing reads.
In <em>the 18th Annual International Conference on Research in Computational Molecular Biology (RECOMB, Pittsburgh PA, USA, 2014)</em>, volume 8394 of LNCS, 237&ndash;249. 2014.
<a href="https://doi.org/10.1007/978-3-319-05269-4_19">doi:10.1007/978-3-319-05269-4_19</a>.</dd>
<dt>46.</dt>
<dd>Cedric Chauve*, Murray Patterson*, and Ashok Rajaraman*.
Hypergraph covering problems motivated by genome assembly questions.
In <em>the International Workshop on Combinatorial Algorithms (IWOCA, Rouen, France, 2013)</em>, volume 8288 of LNCS, 428&ndash;432. 2013.
<a href="https://doi.org/10.1007/978-3-642-45278-9_37">doi:10.1007/978-3-642-45278-9_37</a>.</dd>
<dt>47.</dt>
<dd>Cedric Chauve*, J<span class="bibtex-protected">á</span>n Ma<span class="bibtex-protected">ň</span>uch*, Murray Patterson*, and Roland Wittler*.
Tractability results for the consecutive-ones property with multiplicity.
In <em>the 22nd Annual Symposium on Combinatorial Pattern Matching (CPM, Palermo, Italy, 2011)</em>, volume 6661 of LNCS, 90&ndash;103. 2011.
<a href="https://doi.org/10.1007/978-3-642-21458-5_10">doi:10.1007/978-3-642-21458-5_10</a>.</dd>
<dt>48.</dt>
<dd>J<span class="bibtex-protected">á</span>n Ma<span class="bibtex-protected">ň</span>uch*, Murray Patterson*, and Arvind Gupta.
Towards a characterization of the generalized cladistic character compatibility problem for non-branching character trees.
In <em>the 7th International Symposium on Bioinformatics Research and Applications (ISBRA, Changsha, China, 2011)</em>, volume 6674 of LNCS, 440&ndash;451. 2011.</dd>
<dt>49.</dt>
<dd>J<span class="bibtex-protected">á</span>n Ma<span class="bibtex-protected">ň</span>uch*, Murray Patterson*, Sheung-Hung Poon*, and Chris Thachuk*.
Complexity of finding non-planar rectilinear drawings of graphs.
In <em>the 18th International Symposium on Graph Drawing (GD, Konstanz, Germany, 2010)</em>, volume 6502 of LNCS, 305&ndash;316. 2010.
<a href="https://doi.org/10.1007/978-3-642-18469-7_28">doi:10.1007/978-3-642-18469-7_28</a>.</dd>
<dt>50.</dt>
<dd>Cedric Chauve*, J<span class="bibtex-protected">á</span>n Ma<span class="bibtex-protected">ň</span>uch*, and Murray Patterson*.
On the gapped consecutive-ones property.
In <em>the European Conference on Combinatorics, Graph Theory and Applications (EuroComb, Bordeaux, France, 2009)</em>, volume&nbsp;34 of ENDM, 121&ndash;125. 2009.
<a href="https://doi.org/10.1016/j.endm.2009.07.020">doi:10.1016/j.endm.2009.07.020</a>.</dd>
<dt>51.</dt>
<dd>J<span class="bibtex-protected">á</span>n Ma<span class="bibtex-protected">ň</span>uch*, Murray Patterson*, and Arvind Gupta.
On the generalised character compatibility problem for non-branching character trees.
In <em>the 15th Annual International Computing and Combinatorics Conference (COCOON, Niagara Falls, USA, 2009)</em>, volume 5609 of LNCS, 268&ndash;276. 2009.
<a href="https://doi.org/10.1007/978-3-642-02882-3_27">doi:10.1007/978-3-642-02882-3_27</a>.</dd>
<dt>52.</dt>
<dd>Murray Patterson, Yongmei Liu, Eugenia Ternovska, and Arvind Gupta.
Grounding for model expansion in <em>k</em>-guarded formulas with inductive definitions.
In <em>the 20th International Joint Conferences on Artificial Intelligence (IJCAI, Hyderabad, India, 2007)</em>, 161&ndash;166. 2007.</dd>
</dl></body></html>

<br><h1>Education</h1><hr><br>

<table><tr>
<td width="15%">2006&mdash;2012</td>
<td>

<b>Doctor of Philosophy</b>, <em>Department of Computer
Science, University of British Columbia</em>, Vancouver, Canada, <em>GPA
&mdash; 4.0/4</em>.

<br>

Advisors: J&aacute;n Man&#780;uch, Cedric Chauve and Arvind Gupta

<!--\\

Thesis title: ``Variants of the consecutive-ones property
motivated by the reconstruction of ancestral species''

-->
</td>
</tr></table>

<table><tr>
<td width="15%">2004&mdash;2006</td>
<td>

<b>Master of Science</b>, <em>School of Computing
Science, Simon Fraser University</em>, Burnaby, British Columbia,
Canada, <em>GPA &mdash; 3.9/4</em>.

<br>

Advisors: Eugenia Ternovska and Arvind Gupta

<!--\\

Thesis title: ``Grounding for model expansion in <em>k</em>-guarded
formulas''

-->
</td>
</tr></table>

<table><tr>
<td width="15%">1999&mdash;2003</td>
<td>

<b>Bachelor of Computer Science (Honours)</b>, <em>Jodrey
School of Computer Science, Acadia University</em>, Wolfville, Nova
Scotia, Canada, <em>GPA &mdash; 3.8/4 (top 5%)</em>.

<br>

Advisors: Raymond J. Spiteri and Jim Diamond

<!--\\

Thesis title: ``Implementing Runge-Kutta solvers in Java''

-->
</td>
</tr></table>

    <!-- end content -->

      <br>
      <br>
      <br>

  </body>
</html>


<!--  LocalWords:  Bannow tel ext mpatterson fairfield edu Gianluca Della Paola
 -->
<!--  LocalWords:  Vedova Bonizzoni phylogenies Ancestrome Kahn Daubin catalyse
 -->
<!--  LocalWords:  ERCIM ABCDE INRIA Laboratoire de Biom Biologie Universit CWI
 -->
<!--  LocalWords:  Tannier Schoenhuth Klau tmgo Algorithmics Milano
 -->
<!--  LocalWords:  Bicocca haplotyping LBBE et Gunnar Bioinformatics
 -->
<!--  LocalWords:  Prolog gsu DBLP
 -->
